index.html

```diff
@@ -11,7 +11,11 @@
   <ul>
     <li>Satvir Sandhu</li>
     <li>Hartej Lehal</li>
+    <li>Lawrence Hebia</li>
```

Comment:
I've approved Oksana's PR request. You'll have to pull from master to update your branch to show her edit so to resolve the merge conflict!